.github/workflows/pr.yaml

```diff
@@ -20,55 +20,61 @@ on:
 env:
   TURBO_SCM_BASE: ${{ github.event.pull_request.base.sha }}
   TURBO_SCM_HEAD: ${{ github.sha }}
+  REGISTRY: image-registry.openshift-image-registry.svc:5000
```

Comment:
this is a reference to your local cluster? how can GH action see this? are you doing an `oc login` somewhere?

Reply:
@nickboldt  This is the registry of the openshift which we are using in the openshift CI https://github.com/openshift/release/pull/61311